many-block/src/main.rs

```diff
@@ -0,0 +1,111 @@
+use chrono::DateTime;
+use clap::Parser;
+use many::server::module::blockchain::{BlockArgs, BlockReturns, InfoReturns};
+use many::server::ManyUrl;
+use many::types::blockchain::SingleBlockQuery;
+use many::types::identity::CoseKeyIdentity;
+use many::Identity;
+use std::time::SystemTime;
+use tabled::Tabled;
+
+#[derive(Parser)]
+struct Opts {
+    /// The server to list the blocks from.
+    server: ManyUrl,
+
+    /// Number of blocks to query (default to 30).
+    #[clap(long, default_value = "30")]
+    count: u64,
+
+    /// Max Height. By default, this will be the last block (calling blockchain.info).
+    #[clap(long)]
+    max_height: Option<u64>,
```

Comment:
Error if `count >= max_height`

Reply:
It doesn't error. I just clip it. Clarified.